python/cudf/cudf/tests/test_parquet.py

```diff
@@ -316,6 +318,49 @@ def num_row_groups(rows, group_size):
         assert a == b
 
 
+def test_parquet_read_filtered(tmpdir):
```

Comment:
Test looks good generally.  i would recommend breaking up the test either into multiple tests or more preferably [parameterizing the filters](https://docs.pytest.org/en/stable/parametrize.html).  For the failing string equality, we can xfail until arrow 1.0.1 is used in cudf.  For example of xfail and parameterization you can look here:
https://github.com/rapidsai/cudf/blob/branch-0.15/python/cudf/cudf/tests/test_dataframe.py#L1887-L1898

Reply:
Thanks for the feedback, Ben. I've made changes to (1) break down the tests and made one of them (2) parametrized and (3) skipped until we upgrade to pyarrow 1.0.1.

Reply:
Looks good!